### PR TITLE
Add Save as... action to the application

### LINKED ISSUE #74: Change save menu
We should have two actions in the main menu:
- Save
- Save as...

`Save` is disabled by default, and enable if you already saved a first time or working on a loaded workflow.
`Save as...` always opens a dialog for saving in a new file.

### PR DESCRIPTION
**Pull request by martinRenou**
Fixes #74 

The application has now "Save" and "Save as..." actions. The save action is disabled by default, and enabled only if a Workflow has already been loaded from a file, or the save as action has already been processed. The save as action can always be triggered, and will change the current file on which the application is currently writing.

### CODE REVIEW

File: force_wfmanager/wfmanager_task.py [hunk 1]
```diff
@@ -73,36 +79,57 @@ def create_dock_panes(self):
...
+            wildcard='JSON files (*.json)|*.json|'
+        )
+        result = dialog.open()
+
+        if result is not OK:
+            return
+
+        current_file = dialog.path
+
+        if self._write_workflow(current_file) is True:
```
Comment (stefanoborini) [inline force_wfmanager/wfmanager_task.py]: ==

File: force_wfmanager/wfmanager_task.py [hunk 2]
```diff
@@ -73,36 +79,57 @@ def create_dock_panes(self):
...
-        if len(self.current_file) != 0:
-            current_file = self.current_file
-        else:
-            dialog = FileDialog(
-                action="save as",
-                default_filename="workflow.json",
-                wildcard='JSON files (*.json)|*.json|'
-            )
-            result = dialog.open()
+        self._write_workflow(self.current_file)
```
Comment (stefanoborini) [inline force_wfmanager/wfmanager_task.py]: reset the filename if it fails.

File: force_wfmanager/wfmanager_task.py [hunk 3]
```diff
@@ -73,36 +79,57 @@ def create_dock_panes(self):
...
-                self.current_file = current_file
         except IOError as e:
             error(
                 None,
                 'Cannot save in the requested file:\n\n{}'.format(
                     str(e)),
                 'Error when saving workflow'
             )
+            return False
+        else:
```
Comment (stefanoborini) [inline force_wfmanager/wfmanager_task.py]: Handle generic exception as well.

File: force_wfmanager/wfmanager_task.py [hunk 4]
```diff
@@ -73,36 +79,57 @@ def create_dock_panes(self):
...
+        Boolean:
+            True if it was a success to write in the file, False otherwise
+        """
+        writer = WorkflowWriter()
         try:
-            with open(current_file, 'w') as output:
+            with open(file_path, 'w') as output:
                 writer.write(self.workflow_m, output)
-                self.current_file = current_file
         except IOError as e:
```
Comment (stefanoborini) [inline force_wfmanager/wfmanager_task.py]: Do some logging.

File: force_wfmanager/wfmanager_task.py (hunk 1 above)
Comment (martinRenou) [inline force_wfmanager/wfmanager_task.py]: Apparently, flake8 disagree with you:
```
./force_wfmanager/wfmanager_task.py:101:47: E712 comparison to True should be 'if cond is True:' or 'if cond:'```

**Pull request merged**